systemFlake.nix

```diff
@@ -5,14 +5,19 @@
 , supportedSystems ? flake-utils-plus.lib.defaultSystems
 , inputs
 
-, nixosConfigurations ? { }
 , sharedExtraArgs ? { }
-, nixosProfiles ? { }
 , channels ? { }
 , channelsConfig ? { }
 , sharedModules ? [ ]
 , sharedOverlays ? [ ]
 
+, nixosConfigurations ? { }
+  # nixosProfiles should be deprecated since profiles aren't nixos specific
+, nixosProfiles ? { }
+, configProfiles ? nixosProfiles
+, defaultConfigOutput ? "nixosConfigurations"
+, defaultConfigBuilder ? null
```

Comment:
I think these should be renamed to hosts, but I'll let you decide first @gytis-ivaskevicius.